srcpkgs/python-clint/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'python-clint'
+pkgname=python-clint
+version=0.5.1
+revision=1
+wrksrc="${pkgname#*-}-$version"
+noarch=yes
+build_style=python-module
+pycompile_module="clint"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-args"
+short_desc="Tools for Python2-based command-line applications"
+maintainer="Emerson Ferreira <me@skrps.me>"
+license="ISC"
+homepage="https://github.com/kennethreitz/${pkgname#*-}"
+distfiles="$homepage/archive/v$version.tar.gz"
+checksum=cc2ac47fca097d6235fd12e70a4984a63c4ea1992eb0cee04fe8ed13c88544cd
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-clint_package() {
+	depends="python3-args"
```

Comment:
is also missing noarch=yes and pycompile_module="clint"

Reply:
I thought `python3-clint_package()` would inherit them from the "parent" template... Should I make this change to the other templates then?

Reply:
it does inherit but it is standard practice to define them again.

i asked that question before as well, just passing the answer along.

Reply:
ok then, I'll modify the other packages to conform to it